### PR TITLE
readme for sd sample notebooks

### PR DESCRIPTION
**Pull request by brandondutra**

### CODE REVIEW

File: tutorials/Machine Learning/structured_data/Readme.ipynb [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    "\n",
+    "<br>\n",
+    "# Structure of the notebook samples\n",
+    "\n",
+    "The notebook samples in this folder first demonstrate running analysis, training, batch prediction, and online prediction 'locally' without using GCP services. This first 'local end to end' notebook writes the output of each state to the local file system. The later notebooks demonstrates one of the four steps using cloud services. \n",
+    "\n",
+    "To speed things up and reduce cost, the cloud service notebooks extracts the results of the previous step from the local notebook. For example, the notebooks that perform cloud prediction require a trained model to exist in GCS. Instead of running training using the ML Engine, the cloud  prediction notebooks copy the trained model that the first local notebook make from the local file system to a location on GCS. This means you only pay for the cloud service the individual notebook demonstrates! Of course, the cost of running the cloud service notebooks is small as the datasets are very small.\n",
```
Comment (qimingj) [inline tutorials/Machine Learning/structured_data/Readme.ipynb]: extra space.

Comment (qimingj) [inline tutorials/Machine Learning/structured_data/Readme.ipynb]: I would not mention any "cost" or "pay". If needed Dinesh can add those.

Comment (brandondutra) [inline tutorials/Machine Learning/structured_data/Readme.ipynb]: done

Comment (brandondutra) [inline tutorials/Machine Learning/structured_data/Readme.ipynb]: good point!

**Pull request merged**